panther_driver/launch/imu.launch

```diff
@@ -4,21 +4,15 @@
 
   #### IMU static transform publisher #######################################
 
-  <group if="$(env PANTHER_HAS_CONFIG)">
-    <node pkg="tf" type="static_transform_publisher" name="imu_link"
-      args="$(env PANTHER_IMU_LOCALIZATION_X) 
-      $(env PANTHER_IMU_LOCALIZATION_Y) 
-      $(env PANTHER_IMU_LOCALIZATION_Z) 
-      $(env PANTHER_IMU_ORIENTATION_R) 
-      $(env PANTHER_IMU_ORIENTATION_P) 
-      $(env PANTHER_IMU_ORIENTATION_Y)
-      base_link imu_link 100" 
-    />
-  </group>
-
-  <group unless="$(env PANTHER_HAS_CONFIG)">
-    <node pkg="tf" type="static_transform_publisher" name="imu_link_default" args="0.168 0.028 0.083 3.14 4.71238898038 0.0 base_link imu_link 100" />
-  </group>
+  <node pkg="tf" type="static_transform_publisher" name="imu_link"
+    args="$(optenv PANTHER_IMU_LOCALIZATION_X 0.168) 
+    $(optenv PANTHER_IMU_LOCALIZATION_Y 0.028) 
+    $(optenv PANTHER_IMU_LOCALIZATION_Z 0.083) 
+    $(optenv PANTHER_IMU_ORIENTATION_R 3.14) 
+    $(optenv PANTHER_IMU_ORIENTATION_P -1.57) 
+    $(optenv PANTHER_IMU_ORIENTATION_Y 0.0) 
+    base_link imu_link 100"
+  />
 
   #### Nodelet manager ######################################################
```

Comment:
We could move IMU parameters to YAML as well I guess

Reply:
2 separate files for ImuFilterNodelet and PhidgetsSpatialNodelet?